common/scala/src/main/scala/whisk/core/WhiskConfig.scala

```diff
@@ -264,9 +264,9 @@ object WhiskConfig extends Logging {
     val actionInvokePerMinuteDefaultLimit = "defaultLimits.actions.invokes.perMinute"
     val actionInvokeConcurrentDefaultLimit = "defaultLimits.actions.invokes.concurrent"
     val triggerFirePerMinuteDefaultLimit = "defaultLimits.triggers.fires.perMinute"
-
+    val actionSequenceLimit = "limits.actions.sequence.max"
     val actionInvokePerMinuteLimit = "limits.actions.invokes.perMinute"
     val actionInvokeConcurrentLimit = "limits.actions.invokes.concurrent"
     val triggerFirePerMinuteLimit = "limits.triggers.fires.perMinute"
-
+    val actionSequenceMaxLimit = Map(actionSequenceLimit -> null)
```

Comment:
Not sure I understand this comment. We do this for a bunch of other properties in the same class.  Like this:

```
val edgeHost = Map(edgeHostName -> null, edgeHostApiPort -> null)
    val consulServer = Map(consulServerHost -> null, consulPort -> null)
    val invokerHosts = Map(invokerHostsList -> null)
    val kafkaHost = Map(kafkaHostName -> null, kafkaHostPort -> null)
    val controllerHost = Map(controllerHostName -> null, controllerHostPort -> null)
    val loadbalancerHost = Map(loadbalancerHostName -> null, loadbalancerHostPort -> null)
```